i18n/zh-cn/docusaurus-plugin-content-docs/current/overview/faq.md

```diff
@@ -735,3 +738,29 @@ public class SetSeataInterceptor implements RequestInterceptor {
 - 检查主键中是否包含逗号。
 
 ---
+
+<h3 id='45'>Q: 45. 为什么namingserver和console打包编译需要JDK25版本？& 为什么namingserver和console没有参与编译打包？ </h3>
+
+1. 目前做了什么变更？
+   - 目前namingserver的目标JDK编译版本被设置为25，并且新增了profile来保证在JDK25环境下才会将namingserver和console加入进编译和打包
+   - ```
+     <!-- profile: onlyBuildOnJDK25+ -->
+             <profile>
+                 <id>JDK25Plus</id>
+                 <activation>
+                     <jdk>[25,)</jdk>
+                 </activation>
+                 <modules>
+                     <module>namingserver</module>
+                     <module>console</module>
+                 </modules>
+             </profile>
+     ```
+   - 若使用JDK25以下的版本，编译打包会默认将namingserver和console模块排除。
+2. 为什么要使用JDK25？
+   - 为了支持 namingserver 和 console 后续版本引入的 Spring AI 依赖，必须将 Spring Boot 和 JDK 版本升级。选择 JDK 25 作为目标版本，不仅满足 Spring AI 的技术要求，也为未来新功能的开发和全新功能的适配奠定基础，确保技术栈的前瞻性和扩展性。
+3. 不使用JDK25可以吗？
+   - 目前这个版本还没有引入必须使用JDK25的新功能，最低还是支持JDK8的。若想继续使用JDK25以下的版本，可以在mvn打包编译的时候手动带上 -PJDK25Plus 选项，这样可以在其他JDK版本环境下编译打包namingserver和console模块
```

Comment:
Isn't this description inaccurate? Spring Boot 2.6 is about to be upgraded to Spring Boot 3 very soon, and even if you add the JDK25Plus parameter, packaging will still fail on JDK 17 or lower.